dcos-versions/1.10.4/dcos-bootstrap/templates/install/run.sh

```diff
@@ -75,7 +75,7 @@ ${dcos_package_storage_uri == "" ? "" : "  package_storage_uri: ${dcos_package_s
 EOF
 curl -o dcos_generate_config.${dcos_version}.sh ${dcos_download_path}
 cp /tmp/ip-detect genconf/.
-cp /tmp/ip-detect-public genconf/.
+cp /tmp/public-ip-detect genconf/.
```

Comment:
the default variable for the ip detect script is actually is ip-detect-public. Changing this will cause it to break unless one overwrites the file name. See https://jira.mesosphere.com/browse/DCOS_OSS-905 for more details if you like.